File: specs/esm/line.js
import * as vg from "@uwdata/vgplot";

await vg.coordinator().exec([
  vg.loadParquet("aapl", "data/stocks.parquet", {where: "Symbol = 'AAPL'"})
]);

export default vg.plot(
  vg.lineY(
    vg.from("aapl"),
    {x: "Date", y: "Close"}
  ),
  vg.width(680),
  vg.height(200)
);

File: docs/public/specs/python/line.py
import vgplot as vg

data = vg.data(aapl=vg.parquet("data/stocks.parquet", where="Symbol = 'AAPL'"))

view = vg.plot(
    vg.line_y(data="aapl", x="Date", y="Close"), vg.width(680), vg.height(200)
)

spec = vg.spec(data, view)
